xchange-ascendex/src/main/java/org/knowm/xchange/ascendex/service/AscendexTradeService.java

```diff
@@ -32,6 +31,20 @@ public String placeLimitOrder(LimitOrder limitOrder) throws IOException {
 
   @Override
   public boolean cancelOrder(CancelOrderParams orderParams) throws IOException {
+    if (orderParams instanceof DefaultCancelOrderByInstrumentAndIdParams) {
```

Comment:
you should type check by interface not their default implementation

please do `orderParams instance of CancelOrderByIdParams && orderParams instanceof CancelOrderByInstrument`

if both are needed

if only one is needed then use `||`